common/v2/services/UnstoppableService/validators.ts

```diff
@@ -0,0 +1,6 @@
+import unstoppableResolution from './unstoppableResolution';
+
+export function isValidDomain(domain: string): boolean {
+  const unstoppableResolver = new unstoppableResolution();
+  return unstoppableResolver.isValidDomain(domain);
+}
```

Comment:
I don't think you really need this service - it doesn't do anything on top of `unstoppableResolver`.